src/sim65/paravirt.c

```diff
@@ -327,5 +327,7 @@ void ParaVirtHooks (CPURegs* Regs)
     Hooks[Regs->PC - PARAVIRT_BASE] (Regs);
 
     /* Simulate RTS */
-    Regs->PC = Pop(Regs) + (Pop(Regs) << 8) + 1;
+    unsigned lo = Pop(Regs);
+    unsigned hi = Pop(Regs);
+    Regs->PC = lo + (hi << 8) + 1;
```

Comment:
Don't overdo the fix.  There needs to be a sequence point _between_ the two `Pop()s`.  Therefore, only the first one needs to be outside the PC expression.

Reply:
BTW this is C99 style.

Reply:
Done.